typescript/sdk/src/ism/HyperlaneIsmFactory.ts

```diff
@@ -324,6 +344,7 @@ export async function moduleCanCertainlyVerify(
       }
     }
   }
+  return false;
```

Comment:
why are we reaching this?

Reply:
no idea, wasn't building otherwise

Reply:
presumably because you added a new module type that the switch case doesn't handle? please diagnose this

Reply:
probably null metadata

Reply:
you didn't resolve this
please do not resolve conversations without a code change or comment explaining why you won't change

Reply:
https://github.com/hyperlane-xyz/hyperlane-monorepo/pull/2747/commits/a00dc4ea73644027148b2b8b8b78c90d9e36f2a8 removed here